src/new/wallet.js

```diff
@@ -1634,25 +1732,35 @@ class HathorWallet extends EventEmitter {
   }
 
   /**
-   * @typedef BaseTransactionResponse
-   * @property {{hash:string, index:number, data:Buffer}[]} inputs
-   * @property {{value:number, script:Buffer, tokenData:number, decodedScript:*}[]} outputs
-   * @property {number} version
-   * @property {number} weight
-   * @property {number} nonce
-   * @property {number} timestamp
-   * @property {string[]} parents
-   * @property {string[]} tokens
-   * @property {string} hash
-   * @property {*} _dataToSignCache
-   */
-
-  /**
-   * @typedef CreateNewTokenResponse
-   * @extends BaseTransactionResponse
-   * @property {string} name
-   * @property {string} symbol
-   */
+   * Create a new token for this wallet
+   *
+   * @param {string} name Name of the token
+   * @param {string} symbol Symbol of the token
+   * @param {OutputValueType} amount Quantity of the token to be minted
+   * @param [options] Options parameters
+   * @param {string} [options.address] address of the minted token
+   * @param {string} [options.changeAddress] address of the change output
+   * @param {boolean} [options.startMiningTx=true] boolean to trigger start mining (default true)
+   * @param {string} [options.pinCode] pin to decrypt xpriv information.
+   *                                   Optional but required if not set in this
+   * @param {boolean} [options.createMint=true] should create mint authority
+   * @param {string} [options.mintAuthorityAddress] the address to send the mint authority created
+   * @param {boolean} [options.allowExternalMintAuthorityAddress=false] allow the mint authority address
+   *                                                                    to be from another wallet
+   * @param {boolean} [options.createMelt=true] should create melt authority
+   * @param {string} [options.meltAuthorityAddress] the address to send the melt authority created
+   * @param {boolean} [options.allowExternalMeltAuthorityAddress=false] allow the melt authority address
+   *                                                                    to be from another wallet
+   * @param {string[]|null} [options.data=null] list of data strings using utf8 encoding to add each as a data script output
+   *
+   * @return {Promise<SendTransaction>}
+   * @memberof HathorWallet
+   * @inner
+   * */
+  async createNewTokenSendTransaction(name, symbol, amount, options = {}) {
+    const transaction = await this.prepareCreateNewToken(name, symbol, amount, options);
+    return new SendTransaction({ storage: this.storage, transaction });
```

Comment:
This line is throwing a warning that was not covered by tests but I thought we already had tests covering token creation, is that correct?